partner_contact_role/views/res_partner_role.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Camptocamp
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="res_partner_role_form_view">
```

Comment:
Missing tree and search views.

Reply:
search view is not needed IMO, as we have one single field `name` and search is working by default w/ it